### ISSUE TITLE
Do we need to handle includes?

### ISSUE THREAD

**Issue by rgardler-msft** [Azure/InnovationEngine#12]
As can be seen in [this PR discussion](https://github.com/Azure/InnovationEngine/pull/6#pullrequestreview-1293866623) there is an INCLUDE mechanism in the docs publishing flow. At present Innovation Engine simply ignores these lines.

Is this acceptable? Do these includes ever have required code that needs to be executed?

This issue is a low priority tracking issue to see if we come across places where this would be problematic. If docs team raise a concern then we need to raise this up and actively investigate.

**Comment by rgardler-msft** [comment]
Observation from @naman-msft "Was in a call with an author who is facing issues in a terraform doc that they are converting, which contains "includes" files. Haven't assigned a pri to this yet since you are triaging but wanted to bring it up as a blocker because as per is written in the issue, If docs team raise a concern then we need to raise this up and actively investigate. They have now"

@naman-msft can you please add a test document to the repo. That way we can get it scheduled. I'm pretty sure that once #236 is done this use case can easily be implemented on top of that.

**Comment by naman-msft** [comment]
Test doc: https://learn.microsoft.com/en-us/azure/container-instances/container-instances-quickstart-terraform
